components/script/layout_image.rs

```diff
@@ -39,8 +47,22 @@ impl FetchResponseListener for LayoutImageContext {
     }
 
     fn process_response_eof(&mut self, response: Result<(), NetworkError>) {
+        // TODO notify_pending_response doesn't use the ResourceFetchTiming
         self.cache.notify_pending_response(self.id,
-                                           FetchResponseMsg::ProcessResponseEOF(response));
+                                           FetchResponseMsg::ProcessResponseEOF(
+                                           response.map(|_| ResourceFetchTiming::new())));
+    }
+
+    fn resource_timing(&mut self) -> &mut ResourceFetchTiming {
+        &mut self.resource_timing
+    }
+
+    fn submit_resource_timing(&self) {
+        let local_name = DOMString::from("other");
```

Comment:
We may want to pass "css" here, since this is initiated from a CSS image. I'm not totally clear on the possible different names we should be using.